weave/guides/tracking/tracing.mdx

```diff
@@ -521,65 +526,68 @@ You can also track class and object methods.
 </Tabs>
 
 ## Viewing Calls
+
 <Tabs>
-    <Tab title="Web App">
-    To view a call in the web app:
-    1. Navigate to your project's "Traces" tab
-    2. Find the call you want to view in the list
-    3. Click on the call to open its details page
-    
-    The details page will show the call's inputs, outputs, runtime, and any additional metadata.
-    
-    ![View Call in Web App](/images/screenshots/basic_call.png)
-    </Tab>
-    <Tab title="Python">
-    To view a call using the Python API, you can use the [`get_call`](/weave/reference/python-sdk/trace/weave_client#method-get_call) method:
+<Tab title="Web App">
+To view a call in the web app:
+1. Navigate to your project's "Traces" tab
```

Comment:
Maybe ignore this for now. In the Models docs we use bold face to refer to UI elements.

```suggestion
1. Navigate to your project's **Traces** tab.
```